uwsgi.ini

```diff
@@ -25,10 +25,24 @@ vacuum = True       # Try to remove all of the generated files/sockets (UNIX soc
 module =kolibri.deployment.default.wsgi:application
 buffer-size = 8192  # needed to support long requests blocks (in lessons, for example)
 wsgi-disable-file-wrapper = true  # needed to fix uwsgi bug https://github.com/unbit/uwsgi/issues/1126
+single-interpreter = true
+die-on-term = true                   ; Shutdown when receiving SIGTERM (default is respawn)
+need-app = true
+
+disable-logging = true               ; Disable built-in logging 
+log-4xx = true                       ; but log 4xx's anyway
+log-5xx = true                       ; and 5xx's
+
+py-callos-afterfork = true           ; allow workers to trap signals
+
+max-requests = 1000                  ; Restart workers after this many requests
+max-worker-lifetime = 3600           ; Restart workers after this many seconds
+worker-reload-mercy = 60             ; How long to wait before forcefully killing workers
```

Comment:
super trivial, but just flagging that we're mixing comment delimiters in the same file (`;` vs `#`)

Reply:
No so trivial, thanks. Fixed